packages/compiler/core/node-host.ts

```diff
@@ -21,7 +21,7 @@ export const NodeHost: CompilerHost = {
   readFile: async (path: string) => createSourceFile(await readFile(path, "utf-8"), path),
   writeFile: (path: string, content: string) => writeFile(path, content, { encoding: "utf-8" }),
   readDir: (path: string) => readdir(path),
-  removeDir: (path: string, options: RemoveDirOptions) => rmdir(path, options),
+  removeDir: (path: string, options: RemoveDirOptions) => rm(path, options),
```

Comment:
Do we need to be concerned that this can now remove a non-directory? Should we rename it accordingly and check implementations for consequences?